demo/config/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: blog
+  labels:
+    name: blog
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: blog
+  template:
+    metadata:
+      labels:
+        name: blog
+    spec:
+      containers:
+      - name: git-sync
+        image: staging-k8s.gcr.io/git-sync-amd64:v2.0.5
```

Comment:
The latest version would be v2.0.6. Any reason why you used .5?

Reply:
Nope, no reason! Other than I think I thought 2.0.5 was the most recent version haha...